dolphinscheduler-dist/release-docs/LICENSE

```diff
@@ -244,7 +244,9 @@ The text of each license is also included at licenses/LICENSE-[project].txt.
     commons-math3 3.1.1: https://mvnrepository.com/artifact/org.apache.commons/commons-math3/3.1.1, Apache 2.0
     commons-net 3.1: https://github.com/apache/commons-net, Apache 2.0
     commons-pool 1.6: https://github.com/apache/commons-pool, Apache 2.0
-    cron-utils 5.0.5: https://mvnrepository.com/artifact/com.cronutils/cron-utils/5.0.5, Apache 2.0
+    cron-utils 9.1.3: https://mvnrepository.com/artifact/com.cronutils/cron-utils/9.1.3, Apache 2.0
+    javax.el 3.0.0: https://mvnrepository.com/artifact/org.glassfish/javax.el/3.0.0, CDDL GPL GPL 2.0
```

Comment:
Move this to their corresponding section, this section is all Apache 2.0 license

Reply:
> Move this to their corresponding section, this section is all Apache 2.0 license

@kezhenxu94  Sorry, I have updated, please check if there are any other questions, ths